.github/workflows/extension_ci.yml

```diff
@@ -190,26 +190,24 @@ jobs:
           password: ${{ secrets.QUAY_PASSWORD_TEMBO }}
 
       - name: Build and push -- Commit
-        # push commit build when not a release
         if: github.event_name != 'release'
         uses: docker/build-push-action@v5
         with:
-          file: ./images/pgmq-pg/Dockerfile
+          file: ./images/pgmq-${{ matrix.pg_version }}/Dockerfile
```

Comment:
Does this line need to change? There is no dir `./images/pgmq-pg16` for example and we should be able to use a single dockerfile, right?